apps/base-docs/tutorials/docs/0_gassless-transactions-with-paymaster.md

```diff
@@ -0,0 +1,616 @@
+
+**Navigate to the** **[Coinbase Developer Platform](https://portal.cdp.coinbase.com/)**:
+
+Create or select your project of choice from the upper left corner of your screen.
+
+![cdp-home.png](../../assets/images/gasless-transaction-on-base/cdp-select-project.png)
+
+**Click on the `Paymaster` tool on the left navigation:** [Paymaster Tool](https://portal.cdp.coinbase.com/products/bundler-and-paymaster)
+
+![cdp-paymaster-tool.png](../../assets/images/gasless-transaction-on-base/cdp-paymaster.png)
+
+**Click on `Configuration` at the top of the screen**
+
+![cdp-paymaster-tool.png](../../assets/images/gasless-transaction-on-base/cdp-config.png)
+
+**Save the RPC URL to your paymaster to your clipboard:** You will need it in your `index.js` file in a later step.
+
+### Allowlist a Sponsorable Contract
+
+Sponsoring transactions are beneficial to easing the onboarding and UX of a decentralized application. As a developer, you want to ensure this is done in the most secure and cost-effective manner. Start by allowlisting a contract of your choice.
+
+**Select Base Mainnet**
+
+From the configuration page, select `Base Mainnet` from the dropdown menu. Then, enable your paymaster by clicking on the toggle button to the right of the screen.
+
+![cdp-project-selection.png](../../assets/images/gasless-transaction-on-base/cdp-select-project.png)
+
+**Allowlist the NFT contract and the mintTo functions:**
+
+Click the `Add` button to add a contract.
+
+Add the following contract: `0x83bd615eb93eE1336acA53e185b03B54fF4A17e8`
+
+The function to enable is `mintTo`
+
+![cdp-allowlist-contracts.png](../../assets/images/gasless-transaction-on-base/cdp-allowlist-contract.png)
+
+:::note Use your own contract
+We will be using this simple NFT deployed at X for our example. Feel free to use a contract of your choice.
+:::
+
+**Global Limits:**
+
+This setting allows you to define the maximum amount of gas or USD that can be sponsored globally across all users. It helps control the total expenditure on gas sponsorship, ensuring that the allocated budget is not exceeded. For example, setting a global limit of $1 means that the Paymaster will sponsor transactions until the total gas cost reaches $1.
+
+Set your global policy to be `$.07` by entering the amount in the text field and clicking the `Save` button.
+
+**Scroll down to the “Per User Limit” section:**
+
+We can set a Paymaster policy that specifies either a specific dollar amount or a number of UserOperations. We can enable a 'limit cycle' that allows this policy to refresh every week. This feature enables applications to sponsor smart wallets on a weekly, daily, or monthly basis, either by amount or by number of UserOperations.
+
+Set the max USD to $0.05 and the max UserOperation to 1 to create a policy with a maximum sponsorship limit of $0.05 and a maximum of 1 UserOperation per user.
```

Comment:
I'd add something here about the Limit cycle, even if they're leaving it at none